Scripts/GameManager2.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameManager2 : MonoBehaviour
+{
+    public GameObject startingPanel;
+    public GameObject closePanel;
+    private bool levelCheck;
+
+    private void Awake()
+    {
+        Time.timeScale = 0;
+    }
+    private void Update()
+    {
+        levelCheck = GameObject.FindGameObjectWithTag("bus").GetComponent<BusController>().levelCompleted;
```

Comment:
FindGameObjectWithTag ve GetComponent update'te kullanmak çok yanlış. Bus sürekli değişmeyeceği, sadece level geçişlerinde değişeceği için sadece level değişikliklerinde bir kere bulunması yeterli, update'te kesinlikle bulunmamalı :)